bundles/org.openhab.binding.fmiweather/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,527 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fmiweather"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Things -->
+	<thing-type id="observation">
+		<label>FMI current weather (observation)</label>
+		<description>Finnish Meteorological Institute (FMI) weather observation</description>
+
+		<channel-groups>
+			<channel-group id="current" typeId="current-group" />
+		</channel-groups>
+
+		<config-description>
+			<parameter name="fmisid" type="text" required="true">
```

Comment:
FMI states that FMISID (source, in Finnish: https://www.ilmatieteenlaitos.fi/saahavainnot) is running number, starting from 100000.

I could make it a number indeed with a minimum value -- I guess it would make sense since we get more user interface validation.

Reply:
Resolved in 72e6e6bf0f68700c0a3c156e4dbd81c581c5995f